.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+maxColumn = 112
```

Comment:
I'm against this change.
  
I don't see why sbt maintainers are using their favorite formatting options in sbt and hence forcing all their contributors to suffer them. Personal formatting preferences are really fit for personal projects.
  
Aside from the fact that I think this configuration file is too big, and we should just go for the default, which is a sentiment I've expressed several times in the Gitter channel, I think that the max column is the worst that can happen to these formatting options.
  
I suggest that we go for the gold maxcolumn standard, 80.  Having it larger than that hinders readability in other devices like mobile phones or laptops. People *do* read codebases in these devices, and even write code in them, and we should take them into account.
  
One of the main reasons why I proposed formatting the sbt sources is because reading horizontal code and scrolling left and right is less than an ideal experience. If we're pushing the limits of the max column option, why reformat sbt at all?
  
Plus, this option doesn't make sense if there's no vertical alignment.